pkg/apiserver/resthandler.go

```diff
@@ -676,6 +676,13 @@ func UpdateResource(r rest.Updater, scope RequestScope, typer runtime.ObjectType
 			}
 		}
 
+		canDelete := func() bool {
```

Comment:
Validation would run after the function is called (and verify any invariants are maintained).  The check we're discussing here - field level auth, would either run right before or right after admission (or parts of admission would run before and parts after) inside this function.